examples/digital_fingerprinting/production/README.md

```diff
@@ -80,6 +97,51 @@ Both scripts are capable of running either a training or inference pipeline for
 | `--tracking_uri` | TEXT | The MLflow tracking URI to connect to the tracking backend. [default: `http://localhost:5000`] |
 | `--help` | | Show this message and exit. |
 
+##### Steps to Run Example Pipeline
+The `/workspace/examples/digital_fingerprinting/fetch_example_data.py` script can be used to fetch the Duo and Azure logs to run the example pipelines.
```

Comment:
I would make this a relative path so that it is the same in both the repo and inside the container, and also match the relative paths in `docs/source/developer_guide/guides/5_digital_fingerprinting.md`
```suggestion
The `examples/digital_fingerprinting/fetch_example_data.py` script can be used to fetch the Duo and Azure logs to run the example pipelines.
```